airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -74,7 +73,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class BigQueryDestination implements Destination {
+public class BigQueryDestination extends StandardSQLNaming implements Destination {
```

Comment:
Why are you using inheritance here instead of composition and injecting that helper in the ctor?

Reply:
I can change that yes